Gemfile

```
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'activerecord', '~> 5.2.1'
+gem 'activerecord', '6.0.0.beta2'
```

@kajisha can you please update again as beta3 was recently released?
```suggestion
gem 'activerecord', '6.0.0.beta3'
```

---

Thanks. I updated beta3 at https://github.com/composite-primary-keys/composite_primary_keys/pull/472/commits/0dc230a7b17cc2ed8041b7636220edaf7867e752